core/src/main/scala/org/apache/spark/sql/delta/metering/DeltaLogging.scala

```diff
@@ -47,9 +52,10 @@ import org.apache.spark.sql.delta.util.JsonUtils
  *  Underneath these functions use the standard usage log reporting defined in
  *  [[com.databricks.spark.util.DatabricksLogging]].
  */
-trait DeltaLogging
-  extends DeltaProgressReporter
-  with DatabricksLogging {
```

Comment:
I thik this import is missing in the change

Reply:
Do you mean `DatabricksLogging`?  I think the imports should all be accounted for.

Reply:
yeah I meant DatabricksLogging